src/Orckestra.Composer.Cart/Api/CartController.cs

```diff
@@ -250,7 +255,8 @@ public virtual async Task<IHttpActionResult> AddLineItem(AddLineItemViewModel re
                 Scope = ComposerContext.Scope,
                 CultureInfo = ComposerContext.CultureInfo,
                 CustomerId = ComposerContext.CustomerId,
-                CartName = CartConfiguration.ShoppingCartName,
+                CartName = EditingOrderProvider.IsEditMode() ? EditingOrderProvider.GetCurrentEditingCartName() : CartConfiguration.ShoppingCartName,
```

Comment:
It makes sense to move reusable selector for cart name int separated method or even separate abstraction like CartContext.GetCartName()
CartContext.GetCartType()`

Reply:
extracted to helper methods inside the controller. Is it ok?